bin/test_exercises.sh

```diff
@@ -25,7 +25,7 @@ function test_or_tests () {
     then
       tests+="s"
     fi
-    printf "%s" "$tests"
+    printf '%s' "${tests}"
```

Comment:
We don't actually need to do this here as we know the variable doesn't contain any special characters, but it's done to retain consistency with the rest of our variable-containing-printf-statements.